BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Components/TextFieldWithoutActions.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TextFieldWithoutActions.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+class TextFieldWithoutActions: UITextField {
```

Comment:
[suggestion] I think we need a better name for this. What do you say about: `TextFieldActionsDisabled`?